### PR TITLE
Correct wdl recursive import

### PR DESCRIPTION
**Pull request by wshands**
Base infinite loop on valid WDL

### CODE REVIEW

**Comment by garyluu** [comment]
Gonna wait till the other PR is merged before review because of merge conflicts

File: second-import.wdl
Comment (garyluu) [inline second-import.wdl]: merge conflict

Comment (wshands) [reply second-import.wdl]: Fixed

Comment (wshands) [reply second-import.wdl]: Should be fixed now

Comment (denis-yuen) [inline second-import.wdl]: Merge conflict is still here for me, please confirm

Comment (wshands) [reply second-import.wdl]: Fixed

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**
